sktime/tests/_config.py

```diff
@@ -48,15 +48,15 @@
 
 # TODO fix estimators to pass all tests
 EXCLUDED = [
-    'ContractedShapeletTransform',
+    # 'ContractedShapeletTransform',
```

Comment:
please remove all of the commented lines including the `# 'MrSEQLClassifier'` line to clean this up a bit